roles/serverspec/tasks/main.yml

```diff
@@ -0,0 +1,46 @@
+---
+- name: install ruby packages
+  apt: pkg={{ item }}
+  with_items:
+    - ruby1.9.1-full
+
+- name: install ruby gems
+  shell: GEM_PATH=/opt/sensu/embedded/lib/ruby/gems/2.0.0 /opt/sensu/embedded/bin/gem install rspec-core -v 3.4.0 --no-user-install
+
+- name: install net-ssh gem
+  shell: GEM_PATH=/opt/sensu/embedded/lib/ruby/gems/2.0.0 /opt/sensu/embedded/bin/gem install net-ssh -v 2.9.2 --no-user-install
+
+- name: install specinfra gem
+  shell: GEM_PATH=/opt/sensu/embedded/lib/ruby/gems/2.0.0 /opt/sensu/embedded/bin/gem install specinfra -v 2.50.4 --no-user-install
+
+- name: install serverspec gem
+  shell: GEM_PATH=/opt/sensu/embedded/lib/ruby/gems/2.0.0 /opt/sensu/embedded/bin/gem install serverspec -v 2.29.1 --no-user-install
+
+
+- name: install serverspec-extended-types gem
+  shell: GEM_PATH=/opt/sensu/embedded/lib/ruby/gems/2.0.0 /opt/sensu/embedded/bin/gem install serverspec-extended-types -v 0.0.3 --no-user-install
```

Comment:
Good idea. I was just so happy when it finally worked that I jumped the gun a bit.